modules/apps/commerce/commerce-channel-web/src/main/resources/META-INF/resources/commerce_channel/general.jsp

```diff
@@ -149,17 +149,16 @@ Map<String, String> contextParams = HashMapBuilder.<String, String>put(
 
 				<div class="mb-4" id="autocomplete-root"></div>
 
-				<aui:select label="price-type" name="priceDisplayType">
-
-					<%
-					String priceDisplayType = commerceChannel.getPriceDisplayType();
-					%>
+				<%
+				String priceDisplayType = commerceChannel.getPriceDisplayType();
+				%>
 
+				<aui:select label="price-type" name="priceDisplayType" value="<%= priceDisplayType %>">
 					<aui:option label="net-price" selected="<%= priceDisplayType.equals(CommercePricingConstants.TAX_EXCLUDED_FROM_PRICE) %>" value="<%= CommercePricingConstants.TAX_EXCLUDED_FROM_PRICE %>" />
 					<aui:option label="gross-price" selected="<%= priceDisplayType.equals(CommercePricingConstants.TAX_INCLUDED_IN_PRICE) %>" value="<%= CommercePricingConstants.TAX_INCLUDED_IN_PRICE %>" />
 				</aui:select>
 
-				<aui:select label="discounts-target-price-type" name="discountsTargetNetPrice">
+				<aui:select label="discounts-target-price-type" name="discountsTargetNetPrice" value="<%= commerceChannel.getDiscountsTargetNetPrice() %>">
 					<aui:option label="net-price" selected="<%= commerceChannel.isDiscountsTargetNetPrice() %>" value="true" />
 					<aui:option label="gross-price" selected="<%= commerceChannel.isDiscountsTargetNetPrice() %>" value="false" />
```

Comment:
@brianikim  the issue here is this option, the "selected" check should be
!commerceChannel.isDiscountsTargetNetPrice(), the value you are setting is just an override not the proper solution.